app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInnerState.kt

```diff
@@ -45,7 +45,7 @@ class MessageComposerInnerState(
 
     var attachmentOptionsDisplayed by mutableStateOf(false)
 
-    private fun toEnabled() {
+     private fun toEnabled() {
```

Comment:
There is a small/tiny miss alignment, the extra space at the start of the function 👀